docs, #78: add info to collaborative workflow

k-doering-NOAA opened the pull request:
Adds information about vs code setup to the collaborative workflow (resolves #78)

Bai-Li-NOAA commented on 06-developer-software-guide.Rmd:
````diff
@@ -39,11 +39,82 @@ files, outputs, and build and debug environments. The most popular IDEs
+ install.packages("httpgd")
+ ``` 
+
+A number of optional settings that could be added to the user settings (settings.json) file in vscode to improve the usability of R in VS code.
````
Maybe provide instructions on where the user settings file is located? This link could be helpful: https://code.visualstudio.com/docs/getstarted/settings#_settings-file-locations. I found a few settings.json files in different places on my laptop and the correct settings.json is at %APPDATA%\Roaming\Code\User\settings.json

Bai-Li-NOAA commented on 06-developer-software-guide.Rmd:
```diff
@@ -39,11 +39,82 @@ files, outputs, and build and debug environments. The most popular IDEs
+    "cmake.configureOnOpen": true, 
+    // Set where the rulers are, needed for Rewrap
+    "editor.rulers": [
+        80
```
According to the recommended limit line length from [section 7.3](https://noaa-fims.github.io/collaborative_workflow/contributor-guidelines.html#coding-good-practices), do we want to use 72 instead of 80 here?

k-doering-NOAA replied:
oh, absolutely, good point! Time to modify my own settings.json :)

Bai-Li-NOAA reviewed (commented):
This looks great! I was able to modify the settings.json following the instructions. Let me know if you have any questions about the minor suggestions.

k-doering-NOAA commented:
Thanks for reviewing @Bai-Li-NOAA ! I added a few sentences to address the points you brought up. I also added a few more "tips", based on things that came up while I was setting up VS code on my computer yesterday- If you have time to take a look at the latest (shouldn't take long to review), that would be great! Feel free to merge in if you are happy with it or suggest edits. I am done setting up so shouldn't add anything else right now :)
